packages/ui-components/src/components/absolute-time-picker/absolute-time-picker.tsx

```diff
@@ -48,24 +49,39 @@ export class KvAbsoluteTimePicker implements IAbsoluteTimePicker, IAbsoluteTimeP
 	/** @inheritdoc */
 	@Event() relativeTimeConfigChange: EventEmitter<IAbsoluteSelectedRangeDates>;
 
+	/** From and to input values inserted by the user on the input */
 	@State() fromInputValue: string = '';
 	@State() toInputValue: string = '';
+	/** Shared hovered date between calendars */
 	@State() hoveredDate: string = '';
+	/**  First calendar displayed month (Second calendar is displayedMonth + 1) */
 	@State() displayedMonth: dayjs.Dayjs = fromDateInput(this.initialDate) ?? fromDateInput(new Date());
+	/** Used to force focus the from and to input */
 	@State() fromInputFocused: boolean = true;
 	@State() toInputFocused: boolean = false;
+	/**
+	 * Used to define when the input dates came from the relative time picker where the input
+	 * can be an absolute date or a text (ex: last 24 hours)
+	 */
 	@State() inputMode: ERelativeTimeInputMode = ERelativeTimeInputMode.Date;
+	/** Calendars max and minimum dates */
 	@State() maxDate: string = '';
 	@State() minDate: string = '';
 
 	@Watch('selectedRangeDates')
 	handleSelectedRangeDatesChange(value: string[] = []) {
-		// Reset the component state if the selected dates change
 		if (value.length === 0) {
-			this.fromInputValue = '';
-			this.toInputValue = '';
-			this.minDate = '';
-			this.maxDate = '';
+			this.setFromToInputValues('', '');
```

Comment:
I see this a lot in this file. Please choose one of following the approaches:
- Change the `setFromToInputValues` method arguments to be optional and default to an empty string
- Create a method named `resetInputValues` that abstract this trick and give it a name.